sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -295,18 +297,10 @@ def batch_event_handler(event):
     dict
         HTTP response
     """
-    # splited command looks like this:
-    # [
-    #   '',
-    #   'instrument codice ',
-    #   'level l1a ',
-    #   "file_path 'path/filename'",
-    #   "dependency [{'instrument': 'hit', 'level': 'l0', 'version': 'v00-01'}]"
-    #  ]
-    command = event["detail"]["container"]["command"][0].split("--")
+    command = event["detail"]["container"]["command"]
 
     # Get event inputs ready
-    file_path = command[3].replace("file_path '", "").replace("'", "")
+    file_path = command[5]
```

Comment:
Would it be possible to add this to the event information as another field that could be looked up rather than needing to parse a command out?

Reply:
we can't do that because this event is sent by AWS and we can't change that. But if we want, we can send event from within batch job but I think we decided not to do that since instrument may not do same in their L3 code.

Reply:
Ahh, I see. That makes sense to me then that we need to parse this. We can try and think about other options later.